src/explorer/esplora.ts

```diff
@@ -163,8 +163,14 @@ export async function* fetchAndUnblindTxsGenerator(
         continue;
       }
 
-      txids.push(tx.txid);
-      yield unblindTransaction(tx, blindingKeyGetter);
+      try {
+        yield unblindTransaction(tx, blindingKeyGetter);
+        txids.push(tx.txid);
+      } catch (err) {
+        console.error(
```

Comment:
We cannot assume logging capabilities everywhere. I do think would be better to collect blinding errors in a object and returning it at this point

Reply:
so we can eventually handle in clients how to unblind/retry (if any)